### ISSUE TITLE
BUG: remove deprecated astropy usage: ``show_in_notebook``

### ISSUE THREAD

**Issue by bsipocz** [NASA-NAVO/navo-workshop#176]
Not sure which one to recommend, I would go as far as we should use firefly?

```
WARNING: AstropyDeprecationWarning: show_in_notebook() is deprecated as of 6.1 and to create
         interactive tables it is recommended to use dedicated tools like:
         - https://github.com/bloomberg/ipydatagrid
         - https://docs.bokeh.org/en/latest/docs/user_guide/interaction/widgets.html#datatable
         - https://dash.plotly.com/datatable [warnings]
```

**Comment by bsipocz** [comment]
or ditch Tables and go all the way pandas. or manually enforce non-ellipsed table prints with manual scroller. I mean it's a pity that this functionality got removed.

**Comment by bsipocz** [comment]
OK, so this now produces a traceback as the backend has changed in astropy. Fix is easy for avoiding the traceback, but may need some testing if it indeed does what we want.

https://github.com/astropy/astropy/pull/17268

**Comment by bsipocz** [comment]
This has now been fixed with #179
